cpp/src/slice2cpp/Gen.cpp

```diff
@@ -2825,12 +2881,12 @@ Slice::Gen::ProxyVisitor::visitOperation(const OperationPtr& p)
         C << eb;
         C << nl << "catch(...)";
         C << sb;
-        C << nl << "throw ::std::current_exception();";
+        C << nl << "throw ::std::current_exception();"; // TODO: what are doing here?
         C << eb;
```

Comment:
(Unrelated to this PR)
We need a comment here - especially since I removed the corresponding catch in a previous PR and no test failed as a result.